src/csharp/Grpc.Tools.TestData/testAtInPath/msbuildtest.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
nit:  "testAtInPath" feels a bit off in the .NET world where "camelCaseWithNoncapitalInitial" aren't very common?

Reply:
It also doesn't match the test case name "MsBuildTestAtInPath"